Update `getting_started.md` and `README.md`; fix Examples syntax

ariessunfeld opened the pull request:
- Fixed syntax in `classes.py` and `helpers.py` around "Examples" codeblocks in docstrings; readthedocs should now be able to display the example code (fingers crossed).
- Removed Robbins example from `README` and put at the end of `Getting Started` instead
- Tested and cleaned up all code in `getting_started.md` (much was using older API or not passing paths to `data/` files correctly)
- Added shout-out and link to the `Getting Started` documentation at the beginning of the `README`

@cjtu Can you take a look at where the `dem.tif` file is mentioned (in a code block in the `README`) and think about if you want to give a link for users to download this file?

cjtu commented:
Here a low res dem we can include in the package sample data for that example. The output values in the example probably need to be re-run on this new file too [moon_dem.zip](https://github.com/user-attachments/files/20734715/moon_dem.zip)

ariessunfeld commented:
Thanks for the feedback! Standardized to always use ">>>" preceding Python lines; added `moon_dem.tif` to the `sample_data` dictionary and updated the corresponding example in `README`
